Tasks/AzureFileCopy/task.json

```diff
@@ -58,6 +60,7 @@
         },
         {
             "name": "ConnectedServiceNameARM",
+            "aliases": ["azureSubscription"],
```

Comment:
how does it work when there is a conflict in alias resolution

Reply:
It fails when used more than once in the same task. I am working with @RoopeshNair to address this.

Reply:
This is addressed now. We have unique aliases.